src/mon/MonCommands.h

```diff
@@ -869,6 +869,31 @@ COMMAND("osd pool get-quota " \
         "name=pool,type=CephPoolname ",
         "obtain object or byte limits for pool",
         "osd", "r", "cli,rest")
+COMMAND("osd pool application enable " \
```

Comment:
I'm not wedded to it, but we discussed having checked commands for enabling cephfs|rbd|rgw since we know what they should look like and it would be tedious for an admin to typo them. Did you look at that?